dvc/dagascii.py

```diff
@@ -12,6 +16,44 @@
 from grandalf.routing import EdgeViewer
 from grandalf.routing import route_with_lines
 
+from dvc.utils import boxify
+
+
+logger = logging.getLogger(__name__)
+
+
+DEFAULT_PAGER = "less"
+DEFAULT_PAGER_FORMATTED = "{} --chop-long-lines --clear-screen".format(
+    DEFAULT_PAGER
+)
+DVC_PAGER_VAR_NAME = "DVC_PAGER"
+DVC_PAGER_INFO = """\
+Less command line tool is missing.
+Install & add less tool to your PATH env. var. to automatically send output \
+to pager.
+Also, you can override default pager via {} env. var.
+""".format(
+    DVC_PAGER_VAR_NAME
+)
+
+
+def find_pager(pager_cmd, pager_cmd_with_format, pager_env_name):
+    if sys.stdout.isatty():
+        less_found = find_executable(pager_cmd) is not None
+        if less_found:
+            pager_cmd = os.getenv(pager_env_name, pager_cmd_with_format)
+
+            def less_pager(text):
+                return pydoc.tempfilepager(pydoc.plain(text), pager_cmd)
+
+            pager = less_pager
+        else:
+            logger.info(boxify(DVC_PAGER_INFO, border_color="yellow"))
+            pager = pydoc.plainpager
+    else:
+        pager = pydoc.plainpager
+    return pager
```

Comment:
Let's reformat this a bit to lower the complexity of embedded ifs. E.g.
```
if not sys.stdout.isatty():
    return pydoc.plainpager

if find_executable(pager_cmd):
    ...
    return less_pager

logger.info(...)
return pydoc.plainpager
```

Reply:
done and thanks!